tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,9 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(items) == 0
+    assert item_b not in items
```

Comment:
The first assert is great here, but the second one is unneeded. If the length of the items is 0, we can assume that `item_b` is not in `items`.